README.md

```diff
@@ -72,6 +72,9 @@ An example cross platform middle tier api is provided as part of the solution, i
 
 7. JQ
     - [Download and install JQ](https://stedolan.github.io/jq/download/)
+8. Visual Studio
```

Comment:
I would change this a bit. I would recommend VSTS as the load testing tool: https://docs.microsoft.com/en-us/vsts/load-test/get-started-simple-cloud-load-test?view=vsts We can discuss the many benefits of why to do this. Visual Studio is still a dependency, but only if you want to edit the web tests. Could you add some text to clarify that? Something like 8. Visual Studio (only a pre-req if editing web tests)

Reply:
I found that running load tests on VSTS Portal was limited for what we needed, so I have been running all the tests in Visual Studio. However, Visual Studio still gives the option to run load tests either locally and in VSTS (you're able to connect to your VSTS account to run the load test). https://docs.microsoft.com/en-us/vsts/load-test/getting-started-with-performance-testing?view=vsts

Reply:
I made a new change to the ReadMe that mentions how Visual Studio is used to edit and to execute tests, but giving the option to either run them locally or in the cloud through VSTS. Let me know what you think!